azure-pipelines.yml

```diff
@@ -225,7 +225,7 @@ stages:
                 $env:ARM_SUBSCRIPTION_ID="$(azure_subscription_id)"
                 cd $(Pipeline.Workspace)\drop_terraform\environments\dev
                 terraform init
-                terraform plan -no-color -input=false -var="HMAC_SIGNING_KEY=$(HMAC_SIGNING_KEY)" -var="API_KEYS=$(API_KEYS)" -var="GOV_NOTIFY_API_KEY=$(GOV_NOTIFY_API_KEY)" -var="USER_ROLES_CHANGED_EMAIL_TEMPLATE_ID=$(USER_ROLES_CHANGED_EMAIL_TEMPLATE_ID)"
+                terraform plan -no-color -input=false -var="HMAC_SIGNING_KEY=$(HMAC_SIGNING_KEY)" -var="API_KEYS=$(API_KEYS)" -var="GOV_NOTIFY_API_KEY=$(GOV_NOTIFY_API_KEY)" -var="USER_ROLES_CHANGED_EMAIL_TEMPLATE_ID=$(USER_ROLES_CHANGED_EMAIL_TEMPLATE_ID)" -var="BOOKING_MADE_EMAIL_TEMPLATE_ID=$(BOOKING_MADE_EMAIL_TEMPLATE_ID)" -var="BOOKING_MADE_SMS_TEMPLATE_ID=$(BOOKING_MADE_SMS_TEMPLATE_ID)"
```

Comment:
Are `BOOKING_MADE_EMAIL_TEMPLATE_ID` and `BOOKING_MADE_SMS_TEMPLATE_ID` secrets? If not then they don't need to be added here as they can be passed in via pipeline variables as normal. Only secrets need to be passed into the terraform tasks like this to allow for secret masking in the logs etc. 

I also merged a PR yesterday to remove `USER_ROLES_CHANGED_EMAIL_TEMPLATE_ID` as again as it not secret we treat it as a normal variable and it doesn't need to be explicitly mapped.